src/main/java/ru/practicum/shareit/item/repository/ItemRepository.java

```diff
@@ -0,0 +1,19 @@
+package ru.practicum.shareit.item.repository;
+
+import ru.practicum.shareit.item.model.Item;
+
+import java.util.List;
+
+public interface ItemRepository {
+    Item add(Item item);
+
+    Item update(Item item);
+
+    Item get(long id);
+
+    List<Item> getByUser(long user);
```

Comment:
Здесь поиск идёт через id, поэтому метод и параметр лучше переименовать
```suggestion
    List<Item> getByUserId(long userId);
```